extension_helpers/_setup_helpers.py

```diff
@@ -15,11 +16,36 @@
 from setuptools import find_packages
 from setuptools.config import read_configuration
 
-from ._distutils_helpers import get_compiler
+from ._distutils_helpers import get_compiler, get_distutils_option
 from ._utils import import_file, walk_skip_hidden
 
 __all__ = ['get_extensions', 'pkg_config']
 
+_PEP_384_WARNING = """\
+Not targeting PEP 384 limited API. You can build one binary wheel for each \
+platform to support all Python versions by adding the following to your \
+project's setup.cfg file:
+
+    [bdist_wheel]
+    py_limited_api = cp36
+
+Replace '36' with the lowest Python major and minor version that you wish to \
+support.
+"""
+
+
+def _abi_to_version_info(abi):
+    match = re.fullmatch(r'^cp(\d)(\d)$', abi)
```

Comment:
So this can do `cp38` to `(3, 8)` and `cp310` to `(3, 10)`:
```suggestion
    match = re.fullmatch(r'^cp(\d)(\d+)$', abi)
```